packages/serverless/Dockerfile

```diff
@@ -13,17 +13,22 @@ RUN chown -R appuser:appuser /usr/src/app
 # Switch to the non-root user
 USER appuser
 
-# Copy the application code
+# Copy the application code as the non-root user, including .env.example
 COPY --chown=appuser:appuser . .
 
-# Install dependencies with a frozen lockfile
-RUN yarn install --frozen-lockfile
+USER root
 
-# Build the necessary workspace
-RUN yarn workspace @zkdb/serverless build
+# Install dependencies with the exported environment variables
+RUN --mount=type=secret,id=npm,target=/root/.npmrc yarn install --frozen-lockfile
 
-# Set the working directory for the serverless package
+USER appuser
+
+# Build the necessary workspaces
+RUN yarn workspace @zkdb/storage build && \
+    yarn workspace @zkdb/serverless build
+
+# Set the working directory for the broker service
 WORKDIR /usr/src/app/packages/serverless
 
 # Set the entrypoint to the application
-ENTRYPOINT ["node", "build/src/index.js"]
\ No newline at end of file
+ENTRYPOINT ["node", "build/src/index.js"]
```

Comment:
Each services have their `npm run start` command, please use it first.

Reply:
As I'm checking at the latest commit at `main` branch, there's no `start` script in `package.json`. Can you check it again?

Reply:
Yeah. Use it first, if it doesn't have `start` command, then use your `node` command